nexus-repository-r/src/main/java/org/sonatype/nexus/repository/r/internal/RHostedRecipe.groovy

```diff
@@ -73,7 +73,7 @@ class RHostedRecipe
 
     builder.route(packagesGzMatcher()
```

Comment:
Question: Why is PackagesGz handled differently for hosted implementation?

Reply:
Cause it's the only metadata hosted supports for now. PACKAGES.gz is automatically generated in facet for hosted packages. I already created ticket to implement other ones.